lib/livebook/hubs/team_client.ex

```diff
@@ -22,7 +22,7 @@ defmodule Livebook.Hubs.TeamClient do
     deployment_groups: [],
     app_deployments: [],
     agents: [],
-    app_deployment_statuses: nil
+    app_deployment_statuses: []
```

Comment:
`nil` is intentional actually. It ensures that we always send the first report, as in, this check fails:

https://github.com/livebook-dev/livebook/blob/829b6b2095e65577ee5b17ad43ee29a050d55731/lib/livebook/hubs/team_client.ex#L286-L289